src/Models/Models.jl

```diff
@@ -162,13 +162,25 @@ end
 # Check for NaNs in the first prognostic field (generalizes to prescribed velocities).
 function default_nan_checker(model::OceananigansModels)
     model_fields = prognostic_fields(model)
+
+    if isempty(model_fields) 
+        @warn "The NaNChecker is disabled because this model does not evolve prognostic variables."
```

Comment:
```suggestion
```

Don't think we need a warning...

Reply:
Also, why is this a warning? Is there some reason to be careful about? With no prognostic fields, I'm confused what a NaNChecker would do.